packages/cli/snippets/basic-auth.js

```diff
@@ -4,7 +4,7 @@ const authentication = {
   test: {
     url: 'https://example.com/api/accounts/me.json',
   },
-  connectionLabel: '{{username}}', // Can also be a function, check digest auth below for an example
+  connectionLabel: '{{bundle.authData.username}}', // Can also be a function, check Digest auth below for an example
```

Comment:
This makes the line a little long - I can adjust if necessary, but shortening the comment seemed challenging.